src/components/DpDetails/DpDetails.vue

```diff
@@ -30,6 +32,12 @@ export default {
   },
 
   props: {
+    dataCy: {
+      type: String,
+      required: false,
+      default: 'details'
```

Comment:
a bit redundant, but I have no better Idea

Reply:
Maybe `default: 'detailsElemet'`?

Reply:
I have some Ideas:
```
default: 'detailsElement'
default: 'detailsItem'
default: 'infoDetailsItem'
default: 'detailsSection'
```
@salisdemos

Reply:
maybe just `element` ?

Reply:
Do you mean `detailsElement`? Because just `element` is not enough!
https://github.com/demos-europe/demosplan-ui/pull/895/commits/83fbf1137d51191dde59184bac6dc3d21d66cd08

Reply:
why is `element` not enough? That would lead to strings like `element:detailsWrapper`